pkg/monitor/cluster/arooperatorheartbeat.go

```diff
@@ -0,0 +1,28 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"strconv"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (mon *Monitor) emitAroOperatorHeartbeat(ctx context.Context) error {
+	aroDeployments, err := mon.cli.AppsV1().Deployments("openshift-azure-operator").List(metav1.ListOptions{})
+
+	if err != nil {
+		return err
+	}
+
+	for _, d := range aroDeployments.Items {
+		mon.emitGauge("arooperator.heartbeat", 1, map[string]string{
+			"name":    d.Name,
+			"healthy": strconv.FormatBool(d.Status.UnavailableReplicas == 0),
```

Comment:
consider getting rid of "healthy" and emitting 0 or 1 on the gauge, just like we do on the other metrics called heartbeat (frontend and operator)

Reply:
Done.